Enterprise-CMCS/macpro-mdct-qmr
# CMDCT-3823: updating labels for 2024 measure-eligible pop

### Description
Updating "denominator represent your total measure-eligible population" question to reflect changes noted in [CMDCT-3823](https://jiraent.cms.gov/browse/CMDCT-3823)

---
### How to test 
1. Navigate to any 2024 measure and scroll to the "Does this denominator represent your total measure-eligible population as defined by the Technical Specifications for this measure?" question. Make sure the question and the answers match the new text in the [ticket](https://jiraent.cms.gov/browse/CMDCT-3823)
